code/modules/antagonists/changeling/powers/lesserform.dm

```diff
@@ -6,31 +6,29 @@
 	chemical_cost = 5
 	dna_cost = 1
 	genetic_damage = 3
-	req_human = 1
+	req_human = TRUE
+	power_type = CHANGELING_PURCHASABLE_POWER
 
 //Transform into a monkey.
 /datum/action/changeling/lesserform/sting_action(mob/living/carbon/human/user)
-	var/datum/changeling/changeling = user.mind.changeling
 	if(!user)
-		return 0
+		return FALSE
 	if(user.has_brain_worms())
 		to_chat(user, "<span class='warning'>We cannot perform this ability at the present time!</span>")
```

Comment:
might want to update this comment to be more descriptive, may be out of scope.

Reply:
Gonna leave this alone for now.